### PR TITLE
docs: Updated readme

### PR DESCRIPTION
**Pull request by lentzi90**
**What this PR does / why we need it**: This updates the readme to fix some broken links and keep up with the changes to the code base.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: part of #13 will close the issue when merged to `main`.

**Public facing documentation PR** *(if applicable)* -

**Special notes for reviewer**:
Note that this is just a PR for qa-0.7, not to main.
The part I'm most unsure about is the Kibana SSO. If you know how this works, please check that part extra carefully.

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

**Review by danielharr — APPROVED** [review]
Looks good! Feels nice to refer more documentation to compliantkubernetes.io :slightly_smiling_face:

**Comment by OlleLarsson** [comment]
Just a general note: we do not mention anything about DNS. AFAIK one need to set up x y z dns records in order to successfully install both apps and wc clusters.

File: README.md [hunk 1]
```diff
@@ -142,12 +141,50 @@ Assuming you already have everything needed to install the apps, this is what yo
...
+The user kubeconfig will be configured to use the first namespace by default.
+
+**Kibana** access for the user can be provided either by setting up OIDC or using the internal user database in Elasticsearch:
+- OIDC:
+  - Set `elasticsearch.sso.enabled=true` in `sc-config.yaml`.
+  - Configure any extra role mappings needed under `elasticsearch.extraRoleMappings`.
+- Internal user database:
+  - Log in to Kibana using the admin account.
+  - Create an account for the user.
+  - Give the `kibana_user` and `kubernetes-log-reader` roles to the user.
```
Comment (OlleLarsson) [inline README.md:177]:
```suggestion
  - Give the `kibana_user` and `kubernetes_log_reader` roles to the user.
```

**Review by OlleLarsson — APPROVED** [review]

**Comment by lentzi90** [comment]
Excellent pint about the DNS @OlleLarsson ! I added a little bit about it now. :slightly_smiling_face:

**Review by viktor-f — APPROVED** [review]

**Pull request merged**